### PR TITLE
Updated the links to the Automation controller guide.

### PR DESCRIPTION
**Pull request by rogrange**
Updated document links for the current release. See [AAP-1257](https://issues.redhat.com/browse/AAP-1257) for details.

### CODE REVIEW

File: downstream/assemblies/platform/assembly-multi-machine-install.adoc [hunk 1]
```diff
@@ -30,7 +30,7 @@ include::platform/proc-verify-controller-installation.adoc[leveloffset=3]
 
 * Review _Configuring proxy support for {PlatformName}_ if setting up {ControllerName} with a proxy server.
 * See _Managing usability analytics and data collection from {ControllerName}_ for information on controlling what information you share with Red Hat.
-* See the link:https://docs.ansible.com/ansible-tower/latest/html/quickstart/index.html[Ansible Tower Quick Setup Guide] to learn more about using {ControllerName}.
+* See the link:https://docs.ansible.com/automation-controller/latest/html/quickstart/index.html[Ansible Tower Quick Setup Guide] to learn more about using {ControllerName}.
```
Comment (ariordan-redhat) [inline downstream/assemblies/platform/assembly-multi-machine-install.adoc:33]: The title of the linked doc is now:

Automation Controller Quick Setup Guide

File: downstream/modules/platform/con-cluster-platform-ext-database.adoc [hunk 2]
```diff
@@ -9,7 +9,7 @@ This scenario includes installation of multiple {ControllerName} nodes and an {H
 [NOTE]
 ====
 * Running in a cluster setup requires any database that {ControllerName} uses to be external--PostgreSQL must be installed on a machine that is not one of the primary or secondary tower nodes. When in a redundant setup, the remote PostgreSQL version requirements is *PostgreSQL 12*.
-** See link:https://docs.ansible.com/ansible-tower/3.8.2/html/administration/clustering.html#ag-clustering[Clustering] for more information on configuring a clustered setup.
+** See link:https://docs.ansible.com/automation-controller/latest/html/quickstart/index.html#ag-clustering[Clustering] for more information on configuring a clustered setup.
```
Comment (ariordan-redhat) [inline downstream/modules/platform/con-cluster-platform-ext-database.adoc:12]: I couldn't find a Clustering section in the Quickstart Guide - or is there a plan to move the section from the admin guide to the quickstart guide?

I found the clustering section in the controller guide:
https://docs.ansible.com/automation-controller/latest/html/administration/clustering.html

**Review by ariordan-redhat — COMMENTED** [review]
Just a couple of questions about the linked doc titles

File: downstream/modules/platform/con-cluster-platform-ext-database.adoc (hunk 2 above)
Comment (rogrange) [reply downstream/modules/platform/con-cluster-platform-ext-database.adoc:12]: I will have to ask Chris. These changes are based upon the ticket that had been opened up. [AAP-1527](https://issues.redhat.com/browse/AAP-1257). From our conversation on chat, and the list in the ticket, it appeared to belong to the same URL. We shall see what Chris has to say.

**Pull request merged**